web/src/main/kotlin/no/nav/su/se/bakover/web/routes/behandling/BehandlingRoutes.kt

```diff
@@ -197,10 +199,21 @@ internal fun Route.behandlingRoutes(
                         call.svar(resultat)
                     },
                     {
-                        call.audit("Hentet behandling med id $behandlingId")
+                        call.audit("Hentet behandling med id ${req.behandlingId}")
                         call.respondBytes(it, ContentType.Application.Pdf)
                     }
                 )
+
+        post("$behandlingPath/{behandlingId}/vedtaksutkast") {
+            call.withBehandlingId { behandlingId ->
+                call.withBody<WithFritekstBody> { body ->
+                    handleBrevRequest(call, BrevRequest.MedFritekst(behandlingId, body.fritekst))
```

Comment:
Fungerer dette med authorize? Har du laget tester for å sjekke det?

Reply:
Hvorfor skulle det ikke funke med `authorize`? `handleBrevRequest` er jo bare en lokal funksjon som det kalles til.

Reply:
Aha, det så ikke ut som authorize wrappa denne lenger.